alphalens/tears.py

```diff
@@ -393,6 +395,22 @@ def create_returns_tear_sheet(factor_data,
                                            ylim_percentiles=(5, 95),
                                            ax=ax_quantile_returns_bar_by_group)
 
+    ff_factors = ep.utils.load_portfolio_risk_factors() \
+        .drop(['RF'], axis='columns')
```

Comment:
Can  ep.utils.load_portfolio_risk_factors throw? What happen if you run Alphalens without network connection? I would embed the code in a try catch statement and let Alphalens compute the normal tearsheet if the risk factor data is not available